### ISSUE TITLE
the solution about the low acc in UCF101_01 after loading pretrained-model

### ISSUE THREAD

**Issue by SwordHarry** [kenshohara/3D-ResNets-PyTorch#268]
Hi, every one
when I loaded pretrained-model `r3d50_KMS_200ep.pth`, the train acc is about 96 but val acc is about 48; and my code is
```
python main.py --root_path ./ --video_path ./ucf101_dataset/jpg/ --annotation_path ./ucf101_dataset/data/ucf101_01.json \
--result_path ./results/ --dataset ucf101 --model resnet --model_depth 50 \
--n_classes 101 --batch_size 128 --n_threads 4 --checkpoint 5 \
--inference --tensorboard \
--pretrain_path ./pretrained/r3d50_KMS_200ep.pth --n_pretrain_classes 1139 \
```
the result of acc is very low, but after I set the learning rate, the acc about val is 80+ in the first epoch
```
--learning_rate 0.01
```
so the learning rate in the origin code opt.py is 0.1, 0.1 is not Suitable if we need to load pretrained model !

**Comment by fang520zhu** [comment]
Thank you so much. I just encountered this problem and don't know how to solve it. Thank you so much
